### PR TITLE
Iht 4357 Pre-submission PDF content updates

### PR DESCRIPTION
**Pull request by grantrobson**
All the content marked in red in Andy's document linked on the story have been updated.

### CODE REVIEW

File: app/iht/utils/pdf/MessagesTranslator.scala [hunk 1]
```diff
@@ -26,6 +26,9 @@ object MessagesTranslator {
 
   def getMessagesTextWithParameter(key: String, parameter:String ): String = Messages(key, parameter)
 
+  def getMessagesTextWithTwoParameters(key: String, parameter1:String, parameter2:String ): String =
```
Comment (dmbeer) [inline app/iht/utils/pdf/MessagesTranslator.scala]: Is this not the same as the getMessagesTextWithParameters def

File: conf/pdf/templates/application/debts.xsl [hunk 2]
```diff
@@ -12,6 +12,11 @@
         <fo:block font-family="OpenSans-Bold" font-size="16pt" font-weight="bold" page-break-before="always">
             <xsl:value-of select="i18n:getMessagesText($translator, 'iht.estateReport.debts.owedFromEstate')"/>
         </fo:block>
+
```
Comment (dmbeer) [inline conf/pdf/templates/application/debts.xsl]: remove line breaks

Comment (grantrobson) [inline conf/pdf/templates/application/debts.xsl]: Done

File: app/iht/utils/pdf/MessagesTranslator.scala (hunk 1 above)
Comment (grantrobson) [inline app/iht/utils/pdf/MessagesTranslator.scala]: Yes, I've removed it.

**Pull request merged**